device/android/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+cmake_minimum_required(VERSION 3.8 FATAL_ERROR)
```

Comment:
I would recommend putting all C++ files under
`sherpa/csrc`

The code can be reused if we want an x86 server someday.

Reply:
Also, we only need to maintain the `CMakeLists.txt` inside `sherpa`.

Reply:
Ok, I will put decode_stream.* to sherpa/csrc, but I am afraid we have to keep two CMakeList.txt, that would be more clear.